src/engraving/dom/mmrestrange.cpp

```diff
@@ -87,7 +90,7 @@ PropertyValue MMRestRange::propertyDefault(Pid id) const
 {
     switch (id) {
     case Pid::TEXT_STYLE:
-        return TextStyleType::MMREST_RANGE;
+        return style().styleV(Sid::mmRestRangeTextStyle);
```

Comment:
isn't this already covered by `EngravingObject::propertyDefault`? (i.e. no need to override the method)